### DISCUSSION TITLE
Thanks for moving to Github!

### DISCUSSION THREAD

**Discussion by v3ss0n** [pypy/pypy#4856] [General]
For long time user and supporter of PyPy I was been asking to move to github for many times. Now its time for PyPy to shine!

**Comment by mattip** [comment]
Still waiting for the tidal wave of contributors to swamp our PR reviewer bandwidth...

**Reply by v3ss0n** [reply]
Atleast , Now, promoting PyPy will become easier thanks to its presence on GitHub. In the past, when I recommended PyPy to colleagues or clients, they often questioned my suggestion because it was not on Github. Some even went so far as to 
dismiss PyPy as a "dead project."  Now  people can appreciate the hard work you guys put into this project. This will hopefully change the perception that PyPy is less known or less important .

When I mention PyPy is as ambitious and serious as Linux , people are like are you serious? Even on HN.
